src/algebraic_geometry/ringed_space.lean

```diff
@@ -157,6 +157,41 @@ begin
     { refl } }
 end
 
+/-- This should fire before `basic_open_res`. -/
```

Comment:
I don't think this makes sense as a docstring in mouse-over tooltips. It's really meant is justification for the simp-priority which people will notice when reading the source. Right?
```suggestion
-- This should fire before `basic_open_res`.
```